src/firefly/js/FFEntryPoint.js

```diff
@@ -51,9 +66,10 @@ if (app) {
                 {label:'Example Js Dialog', action:'exampleDialog', type:'COMMAND'}
         ]
     };
-    const props = Object.assign({}, defProps, app);
+    const props = Object.assign(defaults, app);
+    const viewer = app.template === Template.LCV ? LcViewer : FireflyViewer;
```

Comment:
I was thinking just a private map.  I don't think anyone else need to know about this.